### PR TITLE
[doc] add contains operator

### PR DESCRIPTION
**Pull request by Amber1990Zhang**
### What is changed, added or deleted? (Required)

Add contains operator doc and update TOC.

### What is the related PR or file link(s)?

 Code pr: https://github.com/vesoft-inc/nebula/pull/2077

### CODE REVIEW

File: docs/manual-EN/2.query-language/2.functions-and-operators/string-comparison-functions-and-operators.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# String Comparison Functions and Operators
+
+| Name     | Description          |
+|:-----    | :------------------: |
+| CONTAINS | Perform case-insensitive inclusion searching in strings |
+
+* CONTAINS
+
+The `CONTAINS` operator is used to perform case-insensitive matching regardless of location within a string. All non-string data are forced to convert to strings.
```
Comment (whitewum) [inline docs/manual-EN/2.query-language/2.functions-and-operators/string-comparison-functions-and-operators.md:9]: non-string data -> type

File: docs/manual-CN/2.query-language/2.functions-and-operators/string-comparison-functions-and-operators.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# 字符比较函数和运算符
+
+| 名称      | 描述                 |
+|:-----    | :------------------: |
+| CONTAINS | 搜索包含指定字段的字符串，大小写敏感 |
+
+* CONTAINS
+
+`CONTAINS` 运算符用来搜索包含指定字段的字符串，且大小写敏感。所有非字符串数据均强制转换为字符串。
```
Comment (CPWstatic) [inline docs/manual-CN/2.query-language/2.functions-and-operators/string-comparison-functions-and-operators.md:9]:
```
所有非字符串数据均强制转换为字符串。
```

We don't do implicit casting. The `contains` requires string in both left hand side and right hand side.

File: docs/manual-CN/2.query-language/2.functions-and-operators/string-comparison-functions-and-operators.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        YIELD $^.player.name, serve.start_year, serve.end_year, $$.team.name;
+=====================================================================
+| $^.player.name | serve.start_year | serve.end_year | $$.team.name |
+=====================================================================
+| Aron Baynes    | 2001             | 2009           | Warriors     |
+---------------------------------------------------------------------
+```
+
+```ngql
+nebula> GO FROM 107 OVER serve WHERE serve.start_year CONTAINS "07" && \
```
Comment (CPWstatic) [inline docs/manual-CN/2.query-language/2.functions-and-operators/string-comparison-functions-and-operators.md:22]: (string)serve.start_year

File: docs/manual-CN/2.query-language/2.functions-and-operators/string-comparison-functions-and-operators.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+| 名称      | 描述                 |
+|:-----    | :------------------: |
+| CONTAINS | 搜索包含指定字段的字符串，大小写敏感 |
+
+* CONTAINS
+
+`CONTAINS` 运算符用来搜索包含指定字段的字符串，且大小写敏感。`CONTAINS` 运算符左右两侧必须为字符串类型。
+
+```ngql
+nebula> GO FROM 107 OVER serve WHERE $$.team.name CONTAINS "riors" \
```
Comment (laura-ding) [inline docs/manual-CN/2.query-language/2.functions-and-operators/string-comparison-functions-and-operators.md:12]: You can add an example to use  insensitive  case 

```
nebula> GO FROM 107 OVER serve WHERE upper($$.team.name) CONTAINS "WARR" \
               YIELD $^.player.name, serve.start_year, serve.end_year, $$.team.name;
```

Comment (Amber1990Zhang) [reply docs/manual-CN/2.query-language/2.functions-and-operators/string-comparison-functions-and-operators.md:12]: good point 3q

File: docs/manual-CN/2.query-language/2.functions-and-operators/string-comparison-functions-and-operators.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+        YIELD $^.player.name, serve.start_year, serve.end_year, $$.team.name;
+=====================================================================
+| $^.player.name | serve.start_year | serve.end_year | $$.team.name |
+=====================================================================
+| Aron Baynes    | 2001             | 2009           | Warriors     |
+---------------------------------------------------------------------
+
+# 以下语句返回为空。
```
Comment (whitewum) [inline docs/manual-CN/2.query-language/2.functions-and-operators/string-comparison-functions-and-operators.md:20]: use -- as default comment (like sql)
-- 以下语句返回为空

File: docs/manual-EN/2.query-language/2.functions-and-operators/string-comparison-functions-and-operators.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
+# String Comparison Functions and Operators
+
+| Name     | Description          |
+|:-----    | :------------------: |
+| CONTAINS | Perform case-sensitive inclusion searching in strings |
+
+* CONTAINS
+
+The `CONTAINS` operator is used to perform case-sensitive matching regardless of location within a string. All non-string type are forced to convert to strings. The `CONTAINS` operator requires string type in both left and right side.
```
Comment (whitewum) [inline docs/manual-EN/2.query-language/2.functions-and-operators/string-comparison-functions-and-operators.md:9]: R U sure about the "All non-string type are forced to convert to strings"
I don't think so.

File: docs/manual-EN/2.query-language/2.functions-and-operators/string-comparison-functions-and-operators.md [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+```ngql
+nebula> GO FROM 107 OVER serve WHERE $$.team.name CONTAINS "riors" \
+        YIELD $^.player.name, serve.start_year, serve.end_year, $$.team.name;
+=====================================================================
+| $^.player.name | serve.start_year | serve.end_year | $$.team.name |
+=====================================================================
+| Aron Baynes    | 2001             | 2009           | Warriors     |
+---------------------------------------------------------------------
+
+# The follow query returns nothing.
```
Comment (whitewum) [inline docs/manual-EN/2.query-language/2.functions-and-operators/string-comparison-functions-and-operators.md:20]: ditto

What is the default comment

**Pull request merged**